stock_disable_force_availability_button/README.rst

```diff
@@ -0,0 +1,14 @@
+Disable force availability button
+=================================
+
+This module creates a security group, to disable forcing availability
```

Comment:
This module creates a security group that limits the visibility of "Force availability" button on stock operations to those that belong to this group.